App/View Controllers/ImageViewController.swift

```diff
@@ -68,7 +68,6 @@ final class ImageViewController: UIViewController {
 
         let activityViewController = UIActivityViewController(
             activityItems: [
-                imageURL,
```

Comment:
This makes sense to me. If you're curious, the theory is that you pass in all the different representations as activity items and then the destination app picks what they want. So if you're sharing to an image app, it grabs the image, but if you're sharing to a text editor then it would grab the URL. In practice it's a pain, and when destination apps support both images and text they do kinda arbitrary things. Since we have a dedicated "Copy URL" action, I think it makes total sense to leave the bare URL out of the activity items array.

Reply:
Hooray :)

I do understand, I looked into all the available activity types and how you could prepopulate things like the subject for an email app, if sharing that way. But I'm happy that this simple fix will do for this particular scenario.